projects/core/src/user/occ/user.service.ts

```diff
@@ -157,12 +158,31 @@ export class OccUserService {
       .pipe(catchError((error: any) => throwError(error)));
   }
 
-  // TODO: when the endpoint is ready, add implementation here
-  resetPassword(token: string, password: string): Observable<{}> {
-    if (token === undefined || password === undefined) {
-      return of();
-    }
-    return of({});
+  requestForgotPasswordEmail(userEmailAddress: string): Observable<{}> {
+    const url: string = this.getBaseEndPoint() + FORGOT_PASSWORD_ENDPOINT;
```

Comment:
No need to specify `string` in this case, but it's up to you if you want to keep it or not :)

Reply:
It's not my codes :). But, I will remove it.